suite-common/wallet-graph/src/graphReducer.ts

```diff
@@ -49,10 +51,24 @@ const updateSectionPoints = (
 };
 
 export const prepareGraphReducer = createReducerWithExtraDeps(graphInitialState, builder => {
```

Comment:
use just plain `createReducer` we don't need and probably won't need any extra deps

Reply:
03c323a3be9d82529bd3022b1c44ba33a77c0577